zip-bolt-support.rst

```diff
@@ -0,0 +1,334 @@
+::
+
+  ZIP: XXX
+  Title: Add support for Blind Off-chain Lightweight Transactions (Bolt) protocol
+  Authors: J. Ayo Akinyele <ayo@boltlabs.io>
+           Colleen Swanson <swan@boltlabs.io>
+  Credits: Ian Miers <imiers@z.cash>
+           Matthew Green <mgreen@z.cash>
+  Category: Consensus
+  Created: 2019-07-15
+  License: MIT
+
+
+Terminology
+===========
+
+The key words "MUST" and "MUST NOT" in this document are to be interpreted as described in RFC 2119. [#RFC2119]_
+
+Abstract
+========
+
+This proposal specifies three possible approaches for integrating the blind off-chain lightweight transaction (Bolt) protocol [#bolt-paper]_ into Zcash.
+
+Motivation
+==========
+
+Layer 2 protocols like Lightning enable scalable payments for Bitcoin but lack the mechanisms to provide strong privacy guarantees on the payment network. Zcash offers private transactions but currently lacks features that would enable a lightning-style payment channel. This proposal specifies an integration of the Bolt privacy-preserving Lightning protocol on top of Zcash [#bolt-paper]_.
```

Comment:
```suggestion
Layer 2 protocols like Lightning enable scalable payments for Bitcoin but lack the mechanisms to provide strong privacy guarantees on the payment network. Zcash offers private transactions but currently lacks features that would enable a Lightning-style payment channel. This proposal specifies an integration of the Bolt privacy-preserving payment channel protocol on top of Zcash [#bolt-paper]_.
```

Reply:
This assumes too much background about payment channel protocols. A brief description of what they are would be useful. (The Motivation section of a ZIP should be accessible to a general audience that is not steeped in the detail.)